src/lib/routes/admin-api/project/variants.ts

```diff
@@ -96,6 +103,16 @@ export default class VariantsController extends Controller {
             handler: this.overwriteVariants,
             middleware: [
                 openApiService.validPath({
+                    summary:
+                        'Create (overwrite) variants for a feature toggle in all environments',
+                    description: `This overwrites the current variants for the feature toggle specified in the :featureName parameter in all environments.
+
+                    The backend will validate the input for the following invariants
+
+                    * If there are variants, there needs to be at least one variant with ${'`'}weightType: variable${'`'}
+                    * The sum of the weights of variants with ${'`'}weightType: fix${'`'} must be below 1000 (< 1000)
+
+                    The backend will also distribute remaining weight up to 1000 after adding the variants with ${'`'}weightType: fix${'`'} together amongst the variants of ${'`'}weightType: variable${'`'}`,
```

Comment:
If you just want to escape the backticks, you should be able to use backslashes:

```suggestion
                    * If there are variants, there needs to be at least one variant with \`weightType: variable\`
                    * The sum of the weights of variants with \`weightType: fix\` must be strictly less than 1000 (< 1000)

                    The backend will also distribute remaining weight up to 1000 after adding the variants with \`weightType: fix\` together amongst the variants of \`weightType: variable\``,
```